assets/helpers/page/page.js

```diff
@@ -74,13 +75,16 @@ function buildInitialState(
     currencyId,
   };
 
+  // Override the default amounts config with any test participations
+  const amountsWithParticipations = overrideAmountsForParticipations(abParticipations, settings.amounts);
```

Comment:
I find this variable name a bit confusing, maybe `amountsWithParticipationOverrides`?